.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: RELEASE
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  repository_dispatch:
+    types: [release]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  new-version:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check out repo
+        uses: actions/checkout@v2
+      # Node is required for npm
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      # Install and build Docusaurus website
+      - name: Build Docusaurus website
+        run: |
+          yarn 
+          yarn build
+      # https://docusaurus.io/docs/versioning#tagging-a-new-version
+      - name: Tag a new version
+        run: |
+          yarn run docusaurus docs:version ${{ github.event.client_payload.version }}
```

Comment:
Don't we need to commit this to the repo, too?

Reply:
ooh yes!